### ISSUE TITLE
Livereload after Sass (.scss) file change

### ISSUE THREAD

**Issue by samuel-kollat** [angular/angular-cli#815]
1. OS - Windows 10

2. Node version: 5.7.1

3. I installed node-sass (version 3.7.0), renamed test-project.component.css to test-project.component.scss and filled it with:
```
h1 {
  color: blue;
}
```
After running `ng s` everything is allright. But any change to .scss file (e.g. `color: red;`) does not update/reaload a page despite succesful build:

4.
```
file added app\test-project.component.scss___jb_tmp___
file changed app\test-project.component.scss___jb_tmp___
file deleted app\test-project.component.scss
file added app\test-project.component.scss___jb_old___
file deleted app\test-project.component.scss___jb_tmp___
file added app\test-project.component.scss
file deleted app\test-project.component.scss___jb_old___

Build successful - 407ms.

Slowest Trees                                 | Total
----------------------------------------------+---------------------
vendor                                        | 327ms

Slowest Trees (cumulative)                    | Total (avg)
----------------------------------------------+---------------------
vendor (1)                                    | 327ms
Funnel (9)                                    | 34ms (3 ms)
```

Preferably it would be great if a page ptyles were updated without full page reload.

**Comment by yahyaKacem** [comment]
I did the same and everything worked correctly for me, until it just stopped yesterday, all sass files stop being compiled, the only thing that helped me is to delete the node-sass and reinstall it, still don't know what happen, but the cli does compile and reload the page for scss files changes.
maybe the sass compiler is not stable yet since this cli still in beta or something else outside of th cli caused that, anyway try reinstalling node-sass maybe that will help.

**Comment by un33k** [comment]
It is not working for .css either.

$ ng new foobar
$ npm install
$ ng serve

goto: http://localhost:4200
see: **foobar works!**

modify & save:  app/foobar.component.css
noticed: no reload

$ ng -v
angular-cli: 1.0.0-beta.4
node: 5.11.1
os: darwin x64

**Comment by daco** [comment]
I've got the same problem.
angular-cli: 1.0.0-beta.5
node: 4.4.2
os: darwin x64

**Comment by monojack** [comment]
@un33k 
> It is not working for .css either.

this is not related. I am able to reproduce the `css` bug, but editing any `scss` file still reloads the page for me.

**Comment by cescoferraro** [comment]
Same here. Any workarounds?

**Comment by sendilkumarn** [comment]
livereload.js is looking for an import (rel=stylesheet) / style / link property, since we haven't used <app_name>.component.css anywhere, the reload returns false on check.

we can either change it temporarily / suggest a fix at livereload.js

**Comment by developer-hosny** [comment]
temporary solution for css only
you can rename  `app.component.css`   to  `app.component.css.html` and change in component to new file name  `styleUrls: ['app.component.css.html']` and change language mode in you IED to CSS.

**Comment by masaanli** [comment]
Already fixed?

**Comment by Blasz** [comment]
Is there any workarounds for this besides changing the css files to HTML?

**Comment by bruno-serfe** [comment]
Hi people,

Anybody knows if this issue has already a fix or if the fix was added for a future angular-cli build?

thanks,

**Comment by filipesilva** [comment]
Closed as this issue was made obsolete by https://github.com/angular/angular-cli/pull/1455.

**Comment by kilgarenone** [comment]
Hot reload still doesn't seem to trigger from .scss changes. Has this been fixed in 1.0.0-beta.24? I don't see it was fixed in merge #1455 either.
